.lintstagedrc

```diff
@@ -2,7 +2,7 @@
   "*js": [
     "eslint"
   ],
-  "**/*.+(js|jsx|json|yml|yaml|css|scss|md)": [
+  "**/*.+(js|jsx|json|css|scss|md)": [
```

Comment:
yaml formatting seems broken and was corrupting the docker-compose file so removed it from linting